views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -187,8 +189,8 @@ define([
             showReset: false,
             responsive: false,
             applyToMedium: false,
-            width: interaction.object.attr('width'),
-            height: interaction.object.attr('height'),
+            width: interaction.object.attr('width') || defaultSize,
+            height: interaction.object.attr('height') || defaultSize,
```

Comment:
I don't think this is. actually needed.

I mean, it doesn't fix the issue of the `interaction` not having its dimensions, it just hides the problem.

IMO it's better to leave this part as is, and just release the `core-ui` package, updating it on the next release of the extension.